app/javascript/modCenter/moderationArticles.jsx

```diff
@@ -7,35 +7,57 @@ export class ModerationArticles extends Component {
       document.getElementById('mod-index-list').dataset.articles,
     ),
     prevSelectedArticleId: undefined,
-    selectedArticleId: undefined,
   };
 
-  toggleArticle = (id, path) => {
+  toggleArticle = (id, title, path) => {
     const { prevSelectedArticleId } = this.state;
     const selectedArticle = document.getElementById(`article-iframe-${id}`);
+    const selectedDetailsPanel = document.querySelector(
+      `details[data-testid='mod-article-${id}']`
```

Comment:
nitpick (blocking) - since we're using this data attribute for actual app logic and not just testing, I don't think this should be a `data-testid`. Can we convert to an actual `id` on the details element?

Reply:
I added an additional `id` attribute and shifted app logic to it and kept the `data-testid` attribute too since the tests are dependent on it. Is that fine?